shotover/src/runner.rs

```diff
@@ -148,6 +163,13 @@ impl Shotover {
     pub fn run_block(self) -> ! {
         let (trigger_shutdown_tx, trigger_shutdown_rx) = watch::channel(false);
 
+        if self.hotreload_enabled {
+            info!("Starting Shotover with hotreload enabled");
+            //hot reload
```

Comment:
standard format for these kinds of comments is:
`// TODO: hot reload`

Reply:
I'm not sure if this is the correct place for hot reloading logic to go, but for now its as good a place as any, we can move this during implementation if we need to.